modules/ROOT/pages/layers.adoc

```diff
@@ -15,12 +15,11 @@ Map a node to a layer in the application's properties file, or using the UI in t
 
 === Set a Layer with Properties
 
-To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>` in the application's properties file. Note that this is an application property, not a server property.
-
+To automate adding layer(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>`` in the application’s properties file, or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property. You can only use the Runtime Manager UI or Cloudhub deployments.
```

Comment:
this page is for layers but the property says `anypoint.platform.visualizer.tags=<name>, <name>`

Reply:
did you mean to say...`You can only use the Runtime Manager UI **for** Cloudhub deployments."

Reply:
I think we should capture the last line as a Note so we can just drop it when the limitation is fixed. 
"Setting the layer via property file is not currently supported for Mule 4 runtimes"
"You can use the UI only for Cloudhub and RTF deployments"

Reply:
```suggestion
To automate adding layer(s) to a node, use the property `anypoint.platform.visualizer.layer=<name>` in the application’s properties file, or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property. You can only use the Runtime Manager UI for Cloudhub deployments.
```